index.html

```diff
@@ -13,28 +13,154 @@
 
 <body>
   <header>
-    <div class="logo"><a href="#">Narcisse Jemba</a></div>
+    <div class="logo"><a href="#">Narcisse J.</a></div>
     <nav>
       <img src="img/Union.png">
     </nav>
 
   </header>
-  <section class="headline">
+  <section class="headline headline-bg">
     <h1>I’m Alexandra. Glad to see you!</h1>
     <p class="second-text">I’m a software developer! I can help you build a product , feature or website Look through
       some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to
       contact me.</p>
     <div class="third-text">LET’S CONNECT</div>
-     
-      <ul>
-        <li><a href="#"><img src="img/twitter.png"></a></li>
-        <li><a href="#"><img src="img/Linkedin.png"></a></li>
-        <li><a href="#"><img src="img/m.png"></a></li>
-        <li><a href="#"><img src="img/github.png"></a></li>
-        <li><a href="#"><img src="img/fingers.png"></a></li>
-      </ul>
-     
+
+    <ul class="media-icon">
+      <li><a href="#"><img src="img/twitter.png"></a></li>
+      <li><a href="#"><img src="img/Linkedin.png"></a></li>
+      <li><a href="#"><img src="img/m.png"></a></li>
+      <li><a href="#"><img src="img/github.png"></a></li>
+      <li><a href="#"><img src="img/fingers.png"></a></li>
+    </ul>
+
+  </section>
+  <section class="Works">
+    <div class="card">
+      <img src="img/Snapshoot-Portfolio.png" alt="">
+      <h2>Tonic</h2>
+      <ul class="frame2">
+        <li>CANOPY</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>Back End Dev</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="tags">
+        <li class="tag">HTML</li>
+        <li class="tag">CSS</li>
+        <li class="tag">JavaScript</li>
+      </ul>
+      <div class="action">
+        <button>See project</button>
+      </div>
+    </div>
+    <div class="card">
+      <img src="img/Snapshoot-Portfolio1.png" alt="">
+      <h2>Multi-Post Stories</h2>
+      <ul class="frame2">
+        <li>CANOPY</li>
+        <li><img src="img/Counter.png" alt=""></li>
+        <li>Back End Dev</li>
+        <li><img src="img/Counter.png" alt=""></li>
```

Comment:
Please refer back to the previous comment above 👆 about including text in the `alt`.